src/demos/calcite-popover.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
+
+    <title>Calcite App Components: calcite-popover</title>
+
+    <!-- Loads the link and script tags-->
+    <script src="head.js"></script>
+  </head>
+
+  <body>
+    <nav>
+      <calcite-demo-nav page-id="calcite-popover"></calcite-demo-nav>
+    </nav>
+    <main>
+      <section class="example-container">
+        <h1>calcite-popover</h1>
+        <p>
+          see the
+          <a href="https://github.com/Esri/calcite-app-components/tree/master/src/components/calcite-popover/readme.md"
+            >calcite-action readme.</a
+          >
+        </p>
+
+        <style>
```

Comment:
Think this is incomplete. There supposed to be a popover element example here?

Reply:
Yep. its not done, listed in todos